web-console/knikubevirt/expose-secret/expose-secret.md

```diff
@@ -0,0 +1,48 @@
+#Expose Secret, Config Maps and Service Accounts as Disks
+
+Kubernetes recently introduced three new volume types (Secrets, Config Maps and Service Accounts). These volumes can be used as additional VM disks, and allow better integration of Virtual Machines with Kubernetes resources.
+We will try and define where is the best place for the user to handle these disks.
+
+A similar process already exists when attaching Secrets or Config Maps to a deployment config. This happens in the “Environment” section under deployment and deployment config. The Secret/Config Map can be added as an environment variable or as a volume.
+The idea is to be consistent by doing similar processes in the same way across the tool. To support that, we added an “Environment” tab under the VM to handle the attached Secrets, Config Maps and Service Accounts
+
+## Add “Environment” tab to handle the attached Secrets, Config Maps and Services Account
+
+![added environment tab](img/environment-tab.png)
```

Comment:
Can you add pointers to the mockups to show where the user is clicking so that it's a bit easier to follow the flow?